test/BenchmarkMeasurementContainerTest.cpp

```diff
@@ -29,13 +25,11 @@ static auto createWaitLambda(std::chrono::milliseconds waitDuration) {
   };
 }
 
-using namespace std::chrono_literals;
-
 TEST(BenchmarkMeasurementContainerTest, ResultEntry) {
   // There's really no special cases.
   const std::string entryDescriptor{"entry"};
-  // The function should just wait 0.01 seconds.
-  constexpr auto waitTime = 10ms;
+  // The function should just wait 0.1 seconds.
+  constexpr auto waitTime = 100ms;
```

Comment:
Change back to 10ms, this is long enough....

Reply:
Done, see [0fb73ef](https://github.com/ad-freiburg/qlever/pull/1134/commits/0fb73ef2f1539ceec21e1eaa5032db1c22806f40).